src/app/Events/Events.tsx

```diff
@@ -36,33 +36,80 @@
  * SOFTWARE.
  */
 import * as React from 'react';
+import { ServiceContext } from '@app/Shared/Services/Services';
+import { useSubscriptions } from '@app/utils/useSubscriptions';
 import { TargetView } from '@app/TargetView/TargetView';
-import { Card, CardBody, Tab, Tabs } from '@patternfly/react-core';
+import { Card, CardBody, Stack, StackItem, Tab, Tabs } from '@patternfly/react-core';
 import { EventTemplates } from './EventTemplates';
+import { AgentProbeTemplates } from '@app/Agent/AgentProbeTemplates';
+import { AgentLiveProbes } from '@app/Agent/AgentLiveProbes';
 import { EventTypes } from './EventTypes';
+import { concatMap, filter } from 'rxjs/operators';
+import { NO_TARGET } from '@app/Shared/Services/Target.service';
 
 export const Events = () => {
+  const context = React.useContext(ServiceContext);
+  const addSubscription = useSubscriptions();
   const [activeTab, setActiveTab] = React.useState(0);
+  const [enabled, setEnabled] = React.useState(false);
 
-  const handleTabSelect = (evt, idx) => {
-    setActiveTab(idx);
-  };
+  React.useEffect(() => {
+    addSubscription(
+      context.target
+        .target()
+        .pipe(
+          filter((t) => t !== NO_TARGET),
+          concatMap((_) => context.api.getActiveProbes(true))
```

Comment:
`context.api.getActiveProbes` internally calls `context.target.target()`. I think we can remove it here.

Reply:
Edit: This is needed to listen for target change. Added back.